### PR TITLE
Nc news 10

### PR DESCRIPTION
**Pull request by joeglDev**
POST /api/articles/:article_id/comments

### CODE REVIEW

**Comment by joeglDev** [comment]
JUST FIXED THE CO-RUNNING OF ARTICLE_ID CHECK AND SQL QUERY

File: models/models.js [hunk 1]
```diff
@@ -91,5 +93,58 @@ module.exports.selectCommentsByArticleId = (id) => {
...
+
+    .then(({rows : comment}) => {
+      return comment
     });
 };
+
+const insertUser = (username) => {
+  return db.query(
+    ` 
+  INSERT INTO users (username, name) VALUES ($1, $1);`,
```
Comment (k-huggs) [inline models/models.js:141]: are username and name going to be the same thing everytime?

File: __tests__/endpoints.test.js [hunk 2]
```diff
@@ -246,23 +246,111 @@ describe(Endpoints.ALL_COMMENTS_BY_ARTICLE_ID, () => {
...
+  test("returns a http status code of 400 and a error message for if not posted an object with username and body", () => {
+    const newComment = {};
+    return request(app) 
+    .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body: err}) => {
+        expect(err.msg).toBe("Error 400: Malformed request body.")
   })
 });
```
Comment (k-huggs) [inline __tests__/endpoints.test.js:311]: error for if username doesn't exist to be added

**Comment by k-huggs** [comment]
Handle if username doesn't exist error, then merge!

File: models/models.js (hunk 1 above)
Comment (joeglDev) [reply models/models.js:141]: users.name has to be not Null and is not allowed for in the request body

File: __tests__/endpoints.test.js (hunk 2 above)
Comment (joeglDev) [reply __tests__/endpoints.test.js:311]: tests continue to pass for username : undefined and no username property

**Pull request merged**